app/components/edit/contributor_component.html.erb

```diff
@@ -44,13 +44,13 @@
     <div class="cited-section row mb-3">
       <div class="col-md-3 form-check">
         <div class="ps-3">
-          <%= form.radio_button :cited, 'true', class: 'form-check-input' %>
+          <%= form.radio_button :cited, 'true', class: 'form-check-input', data: { action: 'invalid-feedback#hide' } %>
```

Comment:
This indicates that toggling this radio button should clear the invalid feedback for needing a cited contributor.